src/pages/heeyeolKim/Main/Main.js

```diff
@@ -40,28 +42,32 @@ class Main extends React.Component {
             .then(data => {
                 this.setState({feeds: data});
         });
+
+        fetch('/data/storyData.json', {method: 'GET'})
```

Comment:
```suggestion
        fetch('/data/storyData.json')
```
- default 메소드는 GET 이기 때문에 이렇게 생략 가능합니다!